src/api/todos.ts

```diff
@@ -5,4 +5,10 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const postTodos = (todo: Todo) => {
```

Comment:
```suggestion
export const postTodo = (todo: Todo) => {
```
this function allows adding only one todo